bearer/bearer.go

```diff
@@ -236,21 +248,23 @@ func (b Token) VerifySignature() error {
 		signature.NewFromV2(v2.GetSignature()))
 }
 
-// Issuer returns owner.ID associated with the key that signed bearer token.
+// Issuer returns user.ID associated with the key that signed bearer token.
 // To pass node validation it should be owner of requested container.
 //
 // If token is not signed, Issuer returns empty owner ID and false `ok` flag.
 //
 // See also Sign.
-func (b Token) Issuer() (id owner.ID, ok bool) {
+func (b Token) Issuer() (id user.ID, ok bool) {
 	v2 := (acl.BearerToken)(b)
 
 	pub, _ := keys.NewPublicKeyFromBytes(v2.GetSignature().GetKey(), elliptic.P256())
-	if pub == nil {
-		return id, false
+
+	ok = pub != nil
+	if ok {
+		user.IDFromPublicKey(&id, (ecdsa.PublicKey)(*pub), make([]byte, 25))
 	}
 
-	return *owner.NewIDFromPublicKey((*ecdsa.PublicKey)(pub)), true
+	return
```

Comment:
What was wrong with the old implementation?

Reply:
Nothing wrong, there is no `owner.NewIDFromPublicKey` anymore.

Reply:
I've meant `ok = pub != nil` vs `if pub == nil`, but ok not a big deal.